sql/2020/16_Ecommerce/percent_of_mobile_ecommsites_using_each_cmp.sql

```diff
@@ -0,0 +1,32 @@
+#standardSQL
+# 13_17b: % Share of CMPs on mobile eCommerce origins
+SELECT
+  _TABLE_SUFFIX AS client,
+  app,
+  COUNT(DISTINCT url) AS freq,
+  total,
+  COUNT(DISTINCT url) / total AS pct
+FROM
+  `httparchive.technologies.2020_08_01_*`
+JOIN (
+  SELECT
+    _TABLE_SUFFIX,
+    COUNT(DISTINCT url) AS total
+  FROM
+    `httparchive.technologies.2020_08_01_*`
+  WHERE
+     category = 'Ecommerce'
+  GROUP BY
+    _TABLE_SUFFIX
+    )
+USING
+  (_TABLE_SUFFIX)
+WHERE
+  category = 'Cookie compliance'
+  and _TABLE_SUFFIX = 'mobile'
```

Comment:
Why only mobile?

Reply:
@barrypollard - Thought of focusing on mobile for break down purpose. I saw something similar done in Privacy chapter for figure 8 (https://almanac.httparchive.org/en/2020/privacy#fig-8) but after looking at their results sheets and queries I now realize that queries are not device specific. They have just chosen to talk about mobile in the chapter. 

I will delete mobile specific SQLs and create SQLs with device breakdown

Reply:
Yeah if results are the same we advise to not bore readers with double stats but instead have a caveat at the top to say something like "When the results between desktop and mobile are similar, statistics are presented from the mobile dataset." And we normally choose mobile as dataset is bigger, has Lighthouse results (which desktop doesn't have) and we're in a mobile first world nowadays.

But good to run both as you might be surprised to find differences you weren't expecting!